docs/SRS/SRS.tex

```diff
@@ -102,19 +154,24 @@ \subsection{Glossary of All Terms, Including Acronyms, Used by Stakeholders Invo
 a
 
 \section{Relevant Facts and Assumptions}
-a
 
 \subsection{Relevant Facts}
-a
+There is no existing solution 7eam has found that fills a similar role as selecting a Movie, TV Show or Anime based on users' preferences in a group.
+
 
 \subsection{Business Rules}
-a
+Not Applicable
 
 \subsection{Assumptions}
-a
+There are a few assumptions 7eam deems necessary to see regarding Flick Picker's capabilities and what it will not do:
+\begin{itemize}
+	\item Flick Picker will not provide a means to watch the selected show directly through the application
+	\item If the group shares no preferences, then the recommendations will be skipping the preferences of some individual
```

Comment:
Flick Picker will suggest the best fit for the group, which may go against the preferences of a member or multiple members of the group.